coalib/results/Diff.py

```diff
@@ -78,7 +78,7 @@ def from_clang_fixit(cls, fixit, file):
                     fixit.value +
                     oldvalue[endindex:])
         new_file = (file[:fixit.range.start.line-1] +
-                    newvalue.splitlines(True) +
+                    type(file)(newvalue.splitlines(True)) +
                     file[fixit.range.end.line:])
```

Comment:
I'm not that this will fail with arbitrary class. If that class have constructor from list, has subrange and plus operaors everything seems to be OK. Here should be at least assert to validate file's type.